cfme/services/myservice/ssui.py

```diff
@@ -145,16 +145,24 @@ def is_displayed(self):
 
 
 class RetireServiceView(MyServicesView):
-    title = Text(locator='//h4[@id="myModalLabel"]')
+    title = Text(locator='//h4[@class="modal-title"]')
 
-    retire = Button('Yes, Retire Service Now')
+    retire = VersionPicker(
+        {LOWEST: Button('Yes, Retire Service Now'), "5.10": Button('OK')}
```

Comment:
Why not use `Version.lowest()` here?

Reply:
@john-dupuy can you please explain difference between these two things

Reply:
@niyazRedhat turns out they are identical, `LOWEST = Version.lowest()` in `.../miq_version/__init__.py`. I had seen `Version.lowest()` used elsewhere, but I would consider it optional since they are equivalent